src/widgets/category/list/items-view.js

```diff
@@ -84,28 +84,29 @@ module.exports = cdb.core.View.extend({
   _setFilters: function (mdl) {
     var isSelected = mdl.get('selected');
     var filter = this.dataviewModel.filter;
+    var clickedName = mdl.get('name');
 
     if (isSelected) {
       if (filter.rejectedCategories.size() === 0 &&
           filter.acceptedCategories.size() === 0 &&
-          this.dataviewModel.getCount() > 1
+          this.dataviewModel.getSize() > 2
```

Comment:
Not sure what 2 means by looking at this code. Perhaps we can give that number a name by declaring a constant (`MAX_SOMETHING`)?